### ISSUE TITLE
[FEATURE REQUEST] Reuse VDATE value with differ formating

### ISSUE THREAD

**Issue by dariuszkowalski-com** [chhoumann/quickadd#69]
I want to create a note in a folder structure:
- YYYY
   -  MM
      - DD
          FILE-YYYY-MM-DD.md

During a creation process I need only enter DATE ones, then I need to use this with many forms:
{{VDATE:data,YYYY}} - for first level of folders
{{VDATE:data,MM}} - for second
{{VDATE:data:DD}} - for last
{{VDATE:data:YYYY-MM-DD}} - for a file name

Now, 'data' is fixed and is formatted as a first usage.
To solve it I need to ask about Year, Month, Day 
In this scenario, I can't use a natural language like (next week)

**Comment by markusritschel** [comment]
Any news on that? 
I also like the idea of re-using the date. Then, for example, the filename could be defined as  `Meeting - {{VDATE:Date, YYYY-MM-DD}}.md`, and in the template, the VDATE variable `Date` could be re-used with a different formatting, e.g. `{{VDATE:Date, dddd, YYYY-MM-DD}}`.

**Comment by gonced8** [comment]
Would also like to see this.

**Comment by gypsyav** [comment]
I'd love to see this.

**Comment by unspokendeed** [comment]
Likewise - I would appreciate this feature

**Comment by yshalsager** [comment]
+1 for it, really useful for having task added to daily note as in:

`daily/{{VDATE:date, gggg}}/{{VDATE:date, gggg-MM-DD}}.md`

`{{VALUE:task}} {{VALUE:,🔼,⏫,🔺,🔽,⏬️}} 📅 {{VDATE:date, gggg-MM-DD}}`

**Comment by kenNash** [comment]
Definitely +1 here!

Thank you

**Comment by josephtribulat** [comment]
yes please! 🙏 I seldom need the same date format in both the filename and the note's body.

**Comment by pianoman723** [comment]
+1 again!  @chhoumann any update on whether you intend to implement this? Any suggestions for a workaround? Thanks!

**Comment by pianoman723** [comment]
Not sure Christian is paying attention here, but here's my use case.  I have irregularly recurring (pretty complex) project for which I'd use a template and adjust a few things here and there. One of my big pain points is that I have to manually put in due dates every time. But since the due dates are always a function of the project end date (Task 1 due 10 days out, Task 2 due 7 days out, etc etc), there really ought to be a programmatic way to automate this based on the VDATE I set while instantiating the template. I'm sure there's a Javascript way to do this, but that's not in my range of abilities. Any suggestions?
